### PR TITLE
replace hardcoded namespace

### PR DESCRIPTION
**Pull request by pb82**
Remove hardcoded namespace

### CODE REVIEW

**Review by odra — APPROVED** [review]

File: image/tools/lib/component/resources.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ function backup_resource {
             echo '---' > /tmp/${type}.yaml
             for obj in $(oc get ${type} -n ${ns} | tr -s ' ' | cut -d ' ' -f 1 |  tail -n +2); do
                 echo '-' >> /tmp/${type}.yaml
-                echo "$(oc get ${type}/${obj} -n enmasse -o yaml --export | sed 's/^/  /')" >> /tmp/${type}.yaml
+                echo "$(oc get ${type}/${obj} -n ${ns} -o yaml --export | sed 's/^/  /')" >> /tmp/${type}.yaml
                 cat /tmp/${type}.yaml | gzip > ${dest}/archives/${ns}-${type}.yaml.gz
```
Comment (matskiv) [inline image/tools/lib/component/resources.sh:43]: There is still a bug in this part of backup. Not caused by @pb82 change, but I think we can fix it in the same PR.
We are exporting objects in a loop, but each step overrides previously exported object, so we end up just with the last one.

Comment (odra) [reply image/tools/lib/component/resources.sh:43]: should we name it `/tmp/${type}.${obj}` (or something similar)?

Comment (matskiv) [reply image/tools/lib/component/resources.sh:43]: @odra yeah, that's probably cleanest option. Create all the tmp files and then gzip them into ${dest}/archives/${ns}-${type}.yaml.gz together

Comment (odra) [reply image/tools/lib/component/resources.sh:43]: I think we should merge this pr as it is and file a new ticket/pr for this other change which is more complex than just changin the ns var.

Comment (matskiv) [reply image/tools/lib/component/resources.sh:43]: okay, I'll create a JIRA and implement a fix

**Pull request merged**